rtgui/rtwindow.cc

```diff
@@ -282,6 +282,14 @@ RTWindow::RTWindow ()
         show_all ();
 
         bpanel->init (this);
+
+        if (!argv1.empty()) {
+            Thumbnail* thm = cacheMgr->getEntry(argv1);
```

Comment:
perhaps. But explicit is better than implicit in this case. The first version really says "if there is a file to open, let's try to do that". The second says "we expect that there is a file to open, and if we fail to load the thumbnail, we don't open anything"... I prefer version 1 :-)